### PR TITLE
Initial website

### PR DESCRIPTION
**Pull request by jamesmudd**

### CODE REVIEW

**Comment by Stewori** [comment]
I found two rather eye catching typos on the pront page:

"Here is an example of running Python code inside simple Java application."
should be
"code inside a simple Java application"

Footer: "Fundation" should be "Foundation".

That said, I think this should be merged right away. Any further correction can be made on top.
Great solution for the archived sites! Looks good.
Thanks a lot for this work!
From my point of view, just go ahead and merge...

**Comment by jamesmudd** [comment]
Thanks those should be fixed! This PR currently includes the old sites but I stripped out some of the very large content (the podcast mp3 files and the Jython book files). I'm not sure if that's good or not but I don't really want the repository to start off at 300MB. Maybe Git LFS would be useful but i haven't really looked at it.

File: index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * Rapid application development - Python programs are typically 2-10x shorter than the equivalent Java program. This translates directly to increased programmer productivity. The seamless interaction between Python and Java allows developers to freely mix the two languages both during development and in shipping products.
+
+Here is an example of running Python code inside a simple Java application.
+```java
+import org.python.util.PythonInterpreter;
+
+public class JythonHelloWorld {
+  public static void main(String[] args) {
+    try(PythonInterpreter pyInterp = new PythonInterpreter()) {
+      pyInterp.exec("print 'Hello Python World!'");
```
Comment (adamburkegh) [inline index.md]: Perhaps we should use the new print() syntax in our example, ie 

`print('Hello Python World!')`

Even though the old operator syntax still works in 2.7. It's just starting to look declasse

Comment (jamesmudd) [reply index.md]: Yes I agree! I'm too used to using the old print. Hopefully this can be merged soon then you could open a PR?

**Comment by jimbaker** [comment]
LGTM for the first version

**Pull request merged**